CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 
+## [1.5.0]
```

Comment:
Changelogs are usually written by twibiral once it's time to release the next version. I'm not sure if you're advised to write a changelog entry -- @twibiral, should this be here?

Reply:
@chlohal is right, I would prefer if you could remove this. The version number is finalized new version is released and added to the CHANGELOG by a script.